### PR TITLE
1425

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=HtnBWwJ)

### CODE REVIEW

**Comment by yeelali14** [comment]
test

File: README.md [hunk 1]
```diff
@@ -30,3 +30,4 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 #PR
 Lady gaga the queen
+fsfsdfs
```
Comment (yeelali14) [inline README.md:33]: ddd

**Review by yeelali14 — COMMENTED** [review]
kjkj

File: README.md [hunk 2]
```diff
@@ -30,3 +30,4 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 #PR
 Lady gaga the queen
```
Comment (alongalperin-lb) [inline README.md:32]: Happy birthday

File: README.md (hunk 1 above)
Comment (yeelali14) [reply README.md:33]: גג׳ק

**Review by yeelali14 — COMMENTED** [review]
adksd

**Review by ShakedZrihen — APPROVED** [review]

**Pull request closed without merging**